docs/part-1-local-training-and-model-evaluation/chapter-2-adapt-and-move-the-jupyter-notebook-to-python-scripts/index.md

```diff
@@ -633,6 +633,29 @@ def set_seed(seed: int) -> None:
     tf.config.threading.set_intra_op_parallelism_threads(1)
 ```
 
+### Create `README.md`
+
+Finally, create a `README.md` file at the root of the project to describe the
+repository. Feel free to use the following template. As you progress though this
+guide, you can add your notes in the `## Notes` section.
+
+```md title="README.md"
+# MLOps - Planet Classification
+
+[A Guide to MLOps](https://csia-pme.github.io/a-guide-to-mlops/)
+
+## Usage
+
+The code is divided in 4 scripts:
+- `python3 src/prepare.py <raw-dataset-folder> <prepared-dataset-folder>`: prepare the dataset for training
+- `python3 src/train.py <prepared-dataset-folder> <model-folder>`: train the model
+- `python3 src/evaluate.py <model-folder> <prepared-dataset-folder>`: evaluate the model
+- `python3 src/explain.py <model-folder> <raw-dataset-folder>`: explain the model
```

Comment:
This script isn't created in the current version of the guide, am I right?

Should we remove it?

Reply:
Yes you are right